spec/controllers/api/v1/routes_controller_spec.rb

```diff
@@ -97,18 +97,18 @@
   end
 
   describe 'GET show' do
-    it 'returns stops by OnestopID' do
+    it 'returns route by Onestop ID' do
       get :show, id: 'r-9q8y-richmond~dalycity~millbrae'
-      expect_json_types({
+      expect_json_types('route',
         onestop_id: :string,
         geometry: :object,
         name: :string,
         created_at: :date,
         updated_at: :date
-      })
-      expect_json({ onestop_id: -> (onestop_id) {
+      )
+      expect_json('route', onestop_id: -> (onestop_id) {
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Use the `lambda` method for multi-line lambdas.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.